Doc/library/shutil.rst

```diff
@@ -263,8 +261,13 @@ Directory and files operations
 
    If *copy_function* is given, it must be a callable that will be used to copy
    each file. It will be called with the source path and the destination path
-   as arguments. By default, :func:`~shutil.copy2` is used, but any function
-   that supports the same signature (like :func:`~shutil.copy`) can be used.
+   as arguments. By default, :func:`~shutil.copy2` is used, but any function
+   that supports the same signature (like :func:`~shutil.copy`) can be used.
+
+   If *dirs_exist_ok* is false (the default) and *dst* already exists, a
+   :exc:`FileExistsError` is raised. If *dirs_exist_ok* is true, *dst* as well
+   as any parent directory in the path to *dst* will be created if they do
+   not exist already.
```

Comment:
The first sentence, copied from the os.makedirs doc, is correct if the latter is.  And this is all that os.makedirs says about the option.

The second sentence appears wrong in that intermediate are always created as needed.  That needs to be said at the top, as suggested in the previous comment.  I don't think more is needed because it should be clear enough that if
```suggestion
   If *dirs_exist_ok* is false (the default) and *dst* already exists, a
   :exc:`FileExistsError` is raised.
```